core/Inst.hpp

```diff
@@ -32,6 +32,33 @@ namespace olympia
     class Inst {
     public:
 
+        class RenameData{
+            public:
+                void setDestination(uint32_t destination){
+                    dest_ = destination;
+                }
+                void setOriginalDestination(uint32_t destination){
+                    original_dest_ = destination;
+                }
+                using SourceReg = std::pair<uint32_t, bool>;
+                void setSource(SourceReg source){
```

Comment:
Pass in as `const &` to avoid the copy.

Reply:
Addressed in [3eb203c](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/3eb203c761cd11fbebea435284f6d14ba45437a7)